src/dotnet/Chat.Service/AuthorsBackend.cs

```diff
@@ -260,6 +261,42 @@ public virtual async Task<AuthorFull> Upsert(IAuthorsBackend.UpsertCommand comma
         }
     }
 
+    [EventHandler]
+    public virtual async Task OnAvatarChangedEvent(AvatarChangedEvent @event, CancellationToken cancellationToken)
+    {
+        if (Computed.IsInvalidating())
```

Comment:
Is OnAvatarChangedEvent invoked two times: with and without Computed.IsInvalidating() context?
I am quite confused that we have such behavior.

Reply:
@crui3er events are technically commands, so they run in invalidation mode too, yes.

Reply:
@alexyakunin Can we have any benifit from this technical detail? If not, i think it's better to filter calls in invalidation context somewhere under the hood.